web_external/templates/body/layersPanel.pug

```diff
@@ -7,16 +7,39 @@ include ../widgets/legendWidget.pug
   .layersList
     ul.datasets
       each dataset, index in datasets
-        li.dataset&attributes({'m-dataset-id': dataset.get('_id')})
+        - var attributes = {'m-dataset-id': dataset.get('_id')}
+        li.dataset&attributes(attributes)
           if dataset.get('geoError')
-            i.icon-attention.m-remove-dataset-from-layer&attributes({'m-dataset-id': dataset.get('_id')})
+            i.icon-attention.m-remove-dataset-from-layer&attributes(attributes)
           else
-            i.icon-eye.m-toggle-dataset&attributes({'m-dataset-id': dataset.get('_id')})
+            i.icon-eye.icon-button.m-toggle-dataset&attributes(attributes)
+            - var geoRenderType = dataset.getGeoRenderType()
+
             span= dataset.get('name')
             .m-push-right
+              //- Cog icon for geo rendering configuration
+              if geoRenderType === 'geojson' || geoRenderType === 'geojson-timeseries'
+                - var classes = 'icon-cog icon-button m-configure-geo-render'
+                - classes += (dataset.get('displayed') ? ' icon-disabled dataset-in-session' : ' icon-enabled');
+                - classes += ((!dataset.get('displayed') && dataset.get('geoError')) ? ' m-geo-render-error' : '');
+                i(class=classes)&attributes(attributes)
+
+              //- if geoRenderType === 'wms'
```

Comment:
@matthewma7 perhaps we can add a comment here why this code is commented out?

Reply:
Definitely. I'll add some comment. 
In general, since we are not supporting WMS in Minerva currently, the code for WMS is commented out. So, I moved the code but comment it out.